src/logging.cpp

```diff
@@ -1,6 +1,7 @@
 #include "logging.hpp"
 #include <map>
 #include <string>
+#include "static-defines.h"
```

Comment:
ties into "static defines file not needed" but this should not be here

Reply:
As above, mod won't compile without this because of references to ID